Update and rename README.rst to README.md

ffernandoalves opened the pull request:
Trocada a extensão para .md, e adicionado descrição rápida sobre o bot e funcionalidades do projeto.

ffernandoalves committed:
Update and rename README.rst to README.md

Trocada a extensão para .md, e adicionado descrição rápida sobre o bot e funcionalidades do projeto.

MikeDoouglas commented on README.md:
```diff
@@ -0,0 +1,73 @@
+
+# 2 Parte Técnica
+
+O "[DevBot](https://github.com/GrupoDevelopers/DevBot)" está sendo programado em *Python 3.7* utilizando a biblioteca [AIOGram](https://aiogram.readthedocs.io/en/latest/) uma [API de construção de Bot do Telegram](https://core.telegram.org/bots/api). Assim com *[Pyscaffold](https://pyscaffold.org/en/latest/)*, *Git* e *Git Flow*.
```
"Assim com", seria "Assim como" eu acho.

MikeDoouglas commented on README.md:
```diff
@@ -0,0 +1,73 @@
+     
+     4- Execute o script;
+     
+     5- Envie o password pelo chat do Bot que você criou e espere o resultado.
```
Pra executar o bot é só falar o nome dele ("devbot" sem case sensitive) no chat que ele já responde.

MikeDoouglas commented on README.md:
```diff
@@ -0,0 +1,73 @@
+     
+     5- Envie o password pelo chat do Bot que você criou e espere o resultado.
+     
+No atual estado do Bot, ele está somente respondendo de forma a aleatória após receber o password.
```
Não precisa de password na verdade, é só rodar ele e falar 'devbot' pra ele.

MikeDoouglas commented on README.md:
```diff
@@ -0,0 +1,73 @@
+
+Primeiramente, o projeto tem o objetivo, além do supracitado, construir novas habilidades e experiências, desconsiderando o nível que cada um já possui. Pois, muitos podem já ter mais experiências que outros, que é obvio, mas estamos aqui para aprender.
+
+Seria útil para todos nós o compartilhamento de conhecimento e paciência com alguns k.
```
Acho que não ficou muito clara essa frase

ffernandoalves replied:
vou dá uma revisada, mas qual parte mesmo, a ultima ou tudo que ta selecionado ?

MikeDoouglas replied:
Só essa parte "e paciência com alguns k."

daniellecd replied:
Teste.

Z4h4rd commented:
#4 @MikeDoouglas

MikeDoouglas reviewed (commented):
Excelente README, só alguns ajustes pra fazer. Top!

ffernandoalves committed:
Update README.md

Correção nas partes salientadas por @MikeDoouglas e pequenas correções de ortografia e gramática.

ffernandoalves committed:
mudança de palavra, Lei --> Leia
